proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_fi_FI.properties

```
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2018 Velocity Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Olet jo yhdistänyt tälle palvelimelle!
+velocity.error.already-connected-proxy=Olet jo yhdistänyt tälle välityspalvelimelle!
```

"Yhdistänyt" is more of a past tense.
```suggestion
velocity.error.already-connected=Olet jo yhteydessä tälle palvelimelle!
velocity.error.already-connected-proxy=Olet jo yhteydessä tälle välityspalvelimelle!
```